### ISSUE TITLE
Stats

### ISSUE THREAD

**Issue by KittyKatt** [KittyKatt/roulette-linux#6]
Let's see if we can have some more information survive wipes.

 - Shortest uptime
 - Longest uptime

**Comment by loop-de-r00t** [comment]
I see an "easy" way to do this. Before the system is wiped, convert the current uptime back into epoch and compare with the current best times. Replace the old best time with the new best time, then wipe the system

**Comment by KittyKatt** [comment]
Yeah, that's been the thought process in my head on how to do this, just haven't put it to paper yet.
